eZ/Publish/API/Repository/Tests/BaseTest.php

```diff
@@ -158,12 +158,19 @@ protected function getSetupFactory()
     {
         if (null === $this->setupFactory) {
             if (false === isset($_ENV['setupFactory'])) {
-                throw new \ErrorException('Missing mandatory setting $_ENV["setupFactory"].');
+                throw new \ErrorException(
+                    'Missing mandatory setting $_ENV["setupFactory"]. You are trying to run integration tests. Use one of the available phpunit-integration-*.xml configurations'
```

Comment:
As user must have specified test suite I'm usnure if the last part is needed.

Maybe what you are trying to say is:
`Missing mandatory setting $_ENV["setupFactory"], this should normally be set in the relevant phpunit-integration-*.xml file and refer to a setupFactory for the given StorageEngine/SearchEngine in use`

Reply:
> As user must have specified test suite I'm usnure if the last part is needed.

Not really :)
First time I tried to run tests I executed
```
./vendor/bin/phpunit eZ/Publish/API/Repository/Tests/
```
by default PHPUnit takes `phpunit.xml.dist` or `phpunit.xml`

However the message you've provided works for me, changed in the fixup 41e532f